app/app.go

```diff
@@ -666,7 +665,7 @@ func NewApp(
 	app.BatchingKeeper = batchingkeeper.NewKeeper(
 		appCodec,
 		runtime.NewKVStoreService(keys[batchingtypes.StoreKey]),
-		authtypes.NewModuleAddress(batchingtypes.ModuleName).String(),
+		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
```

Comment:
Hm it doesn't look like this is used anywhere since batching doesn't have update params tx. Should we add a quick issue for it?

Reply:
Oh shit 😂 Yeah probably better to remove it until we actually need it